src/netbase.cpp

```diff
@@ -541,58 +523,52 @@ static bool Socks5(const std::string& strDest, int port, const ProxyCredentials
     uint8_t pchRet3[256];
     switch (pchRet2[3])
     {
-        case SOCKS5Atyp::IPV4: recvr = InterruptibleRecv(pchRet3, 4, SOCKS5_RECV_TIMEOUT, hSocket); break;
-        case SOCKS5Atyp::IPV6: recvr = InterruptibleRecv(pchRet3, 16, SOCKS5_RECV_TIMEOUT, hSocket); break;
+        case SOCKS5Atyp::IPV4: recvr = InterruptibleRecv(pchRet3, 4, SOCKS5_RECV_TIMEOUT, sock); break;
+        case SOCKS5Atyp::IPV6: recvr = InterruptibleRecv(pchRet3, 16, SOCKS5_RECV_TIMEOUT, sock); break;
```

Comment:
The diff could be smaller, faster, easier and safer to review without needless renamings mixed into the diff. Maybe open a separate pull for the renamings and the formatting changes.

Reply:
I split this commit in two: variable type change from `SOCKET` to `Sock` + variable rename from `hSocket` to `sock`.

It is good to have the implied convention of naming `SOCKET` variables `hSocket` (old code) or `socket` (new code) and `Sock` variables `sock`.